vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/EmailField.java

```diff
@@ -203,21 +245,48 @@ public EmailField(String label, String initialValue,
     }
 
     /**
-     * Maximum number of characters (in Unicode code points) that the user can
-     * enter.
+     * {@inheritDoc}
+     * <p>
+     * Distinct error messages for email format and different constraints can be
```

Comment:
👍 This addition is excellent – should do a good job of making the i18n method discoverable.